Makefile

```diff
@@ -207,6 +207,7 @@ $(LATEST_TAGS): %@latest: %/Dockerfile %-parent-check
 	@echo "Building [$@] image"
 	@echo
 	cd $* && time $(SHELL) -c "( tar -czh . | docker build ${BUILD_ARGS} $(DBFLAGS_$*) -t $(call docker-tag,$@) --label $(LABEL) - )"
+	docker history $(call docker-tag,$@)
```

Comment:
what is this for?

Reply:
It shows each docker image layer and how much size it take. I think it might be useful. It is good to know how big is your image and which layer takes the most.